openmc/mesh.py

```diff
@@ -210,3 +227,90 @@ def get_mesh_xml(self):
             subelement.text = ' '.join(map(str, self._width))
 
         return element
+
+    def build_cells(self, bc=['reflective'] * 6):
+        """Generates a list of cells which mimic the mesh geometry.
```

Comment:
- [x] for all, though distribcell discussion will continue elsewhere (@ bottom of comments chain right now)